### PR TITLE
Math magicians: components

### PR DESCRIPTION
**Pull request by ngeelani48**
In this Branch Following Functionalities are added:

- Create a new directory called components and Inside components, and create a new `Calculator.js` file.
- Implement the design only (JSX and CSS).

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+      <button className="calc-button" type="button"> 1 </button>
+      <button className="calc-button" type="button"> 2 </button>
+      <button className="calc-button" type="button"> 3 </button>
+      <button className="calc-button-right" type="button"> + </button>
+      <button className="calc-button" id="zero" type="button"> 0 </button>
+      <button className="calc-button" type="button"> . </button>
+      <button className="calc-button-right" type="button"> = </button>
+    </div>
+  </div>
+);
```
Comment (Owusu-Desmond) [inline src/components/Calculator.js]: - [ ] Please form the project requirement, you should make sure that your `Calculator.js` have at least 1 child component :pray: You can read about child and parent components [here](https://bobbyhadz.com/blog/react-pass-data-from-child-to-parent)

**Review by Owusu-Desmond — CHANGES_REQUESTED** [review]
Hi @ngeelani48 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Calculator.js (hunk 1 above)
Comment (ngeelani48) [reply src/components/Calculator.js]: @Owusu-Desmond Thanks for the review 🤝 . I will do it and submit it again 💯 .

File: src/components/Calculator.js [hunk 2]
```diff
@@ -1,30 +1,10 @@
...
-      <button className="calc-button-right" type="button"> + </button>
-      <button className="calc-button" id="zero" type="button"> 0 </button>
-      <button className="calc-button" type="button"> . </button>
-      <button className="calc-button-right" type="button"> = </button>
-    </div>
-  </div>
-);
+const Calculator = () => {
+  return (
+    <CalcDisplay></CalcDisplay>
```
Comment (jasonfelice) [inline src/components/Calculator.js]: - [ ] [Optional] You can self-close this component. Here's how: `<CalcDisplay />`

**Review by jasonfelice — APPROVED** [review]
Hi @ngeelani48, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

<p align="center"></p>

## Highlights
No Linter errors✔️
Descriptive PR ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Calculator.js (hunk 2 above)
Comment (ngeelani48) [reply src/components/Calculator.js]: @jaonfelice Thanks for the review 💯 . Yeah, it was automatically done while fixing linters.

https://github.com/ngeelani48/math-magicians/pull/2/commits/a43f205ac347fbd2c2940e03fd0c8532ff4adcd5#diff-2e862d0b88bdd42eb13f6528bc2ef48a56500daca0350696b337c7f2127f4f0bR4-R6

**Pull request merged**